### PR TITLE
Adding commit policy page to documentation

### PR DESCRIPTION
**Pull request by blrm**

### CODE REVIEW

File: docs/guides/pull_requests.rst [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+Guidelines
+----------
+
+Submitting
+^^^^^^^^^^
+
+* If your code will require the documentation to be updated, include those documentation updates in your PR.
+* Once submitted, request review from the team (via IRC or email).
+* If a reviewer responds with comments/questions, address them promptly to ensure your PR is merged in quickly.
```
Comment (jamesooden) [inline docs/guides/pull_requests.rst]: My southern self feels the need for a "please" before "address"

File: docs/guides/pull_requests.rst [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+merged. Currently, the way we handle this workflow is with the Github integration lgtm.co. The process is
+simple:
+
+1. A developer submits a new Pull Request and asks the team to review.
+2. Once a team member reviews the code and decides it is acceptable, they comment on the PR with the
+   acronym "lgtm" (short for "Looks good to me").
+3. Once two reviewers have commented with "lgtm", the merge button will unlock, and the PR should be merged by
+   the developer who submitted it.
+
+   * The PR should be merged by its submitter so they can make the final decision on its readiness for merging.
```
Comment (jamesooden) [inline docs/guides/pull_requests.rst]: You simplify this to:

This is so they can make the final decision on its readiness for merging.

And add a little asterisk at the end of 3.

**Comment by jamesooden** [comment]
lgtm

**Comment by LandoCalrizzian** [comment]
lgtm

**Pull request merged**